src/integrationTest/groovy/CompilationAndGeneration.groovy

```diff
@@ -18,8 +18,8 @@ public class CompilationAndGeneration {
         ImmutableMultimap.Builder<String, ArrayList<String>> builder = ImmutableMultimap.builder();
         builder.put(DIRECTORIES_ROOT + "subproj/build/classes/generatedSource", ["Subproj.class", "Subproj2.class"])
         builder.put(DIRECTORIES_ROOT + "sub2/build/classes/generatedSource", ["Sub2.class", "Sub22.class"])
-        builder.put(DIRECTORIES_ROOT + "subproj/generated/java", ["Subproj.java", "Subproj2.java"])
-        builder.put(DIRECTORIES_ROOT + "sub2/generated/java", ["Sub2.java", "Sub22.java"])
+        builder.put(DIRECTORIES_ROOT + "subproj/customPath/generated/java", ["Subproj.java", "Subproj2.java"])
```

Comment:
Can you make a "checklist" of tests that _should_ be added but haven't yet? I think that's a fair compromise!